samples/ios-cocoapods-static-framework/settings.gradle.kts

```diff
@@ -0,0 +1,47 @@
+pluginManagement {
+    repositories {
+        mavenLocal()
+        gradlePluginPortal()
+        maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+        google()
+    }
+
+    plugins {
+        val kotlinVersion = extra["kotlin.version"] as String
+        val agpVersion = extra["agp.version"] as String
+        val composeVersion = extra["compose.version"] as String
+
+        kotlin("jvm").version(kotlinVersion)
+        kotlin("multiplatform").version(kotlinVersion)
+        kotlin("android").version(kotlinVersion)
+
+        id("com.android.application").version(agpVersion)
+        id("com.android.library").version(agpVersion)
+
+        id("org.jetbrains.compose").version(composeVersion)
+    }
+}
+
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version("0.4.0")
+}
+
+dependencyResolutionManagement {
+    repositories {
+        mavenLocal()
+        google()
+        mavenCentral()
+        maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+    }
+
+    versionCatalogs {
+        create("moko") {
+            from(files("../../gradle/moko.versions.toml"))
+        }
+    }
+}
+
+rootProject.name = "ios.cocoapods.static.framework"
+
+include(":androidApp")
```

Comment:
why ios cocoapods sample contains android?

Reply:
provided of community sample include android target. I removed android from this sample